fix: Restore GST env vars (#457)
intel-retail/automated-self-checkout: docs_src/dev-tools/environment_variables.md

    |`AGGREGATE` | aggregate branches of the gstreamer pipeline, if any, at the end of the pipeline | "", "gvametaaggregate name=aggregate", "aggregate branch. ! queue" |    
    |`VA_SURFACE` | use video analytics surface from the shared memory if applicable | "", "! "video/x-raw(memory |VASurface)" (GPU only)" |
    |`PARALLEL_PIPELINE` | run pipeline in parallel using the tee branch | "", "! tee name=branch ! queue" |
    |`PARALLEL_AGGRAGATE` | aggregate parallel pipeline results together, paired use with PARALLEL_PIPELINE | "", "! gvametaaggregate name=aggregate ! gvametaconvert name=metaconvert add-empty-results=true ! gvametapublish name=destination file-format=2 file-path=/tmp/results/r$cid_count.jsonl ! fpsdisplaysink video-sink=fakesink sync=true --verbose branch. ! queue !" |
    |`DETECTION_OPTIONS` | extra object detection pipeline instruction parameters | "", "gpu-throughput-streams=4 nireq=4 batch-size=1" |
    |`CLASSIFICATION_OPTIONS` | extra classification pipeline instruction parameters | "", "reclassify-interval=1 batch-size=1 nireq=4 gpu-throughput-streams=4" |

>     |`CPU_SW_DECODER` | ? | "force-sw-decoders=1" |

Reviewer: ```suggestion
    |`CPU_SW_DECODER` | ? | "force-sw-decoders=1" |
```
extra, removed

    |`CPU_SW_DECODER` | force to use software decoder for gst-launch decoding video frames in CPU | "force-sw-decoders=1" |
    |`DECODE` | decoding element instructions for gst-launch to use | Ex: "decode bin force-sw-decoders=1" |
    |`OCR_RECLASSIFY_INTERVAL` | time interval in seconds for OCR classification | Ex: 5 |
    |`BARCODE_RECLASSIFY_INTERVAL` | time interval in seconds for barcode classification | Ex: 5 |

=== "OVMS EVs"
    This list of EVs specifically supports OVMS workloads.

    | Variable | Description | Values |
    |:----|:----|:---|
    |`PIPELINE_PROFILE` | for choosing OVMS workload's pipeline profile to run | use `make list-profiles` to see Values |
    |`RENDER_MODE` | for displaying pipeline and overlay CV metadata | 1, 0 |
    |`STREAM_DENSITY_MODE` | for starting pipeline stream density testing | 1, 0 |
    |`STREAM_DENSITY_FPS` | for setting stream density target fps value | Ex: 15.0 |

>     |`STREAM_DENSITY_INCREMENTS` |for setting incrementing number of pipelines for running stream density| Ex: 1 |

Reviewer: IMO, we probably want to combine this list into common list below together.

=== "Common EVs"

>     This list of EVs specifically supports both profiles.

Reviewer: ```suggestion
    This list of EVs is common for all profiles.
```
